### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Reflow: Two scroll bar appearing for whole page at 400% zoom

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5102]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Reflow;#WCAG1.4.10; #FTP; #ChromiumEdge;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Pre-requisites:**
Set the Display Resolution : 1280*1024.

**Repro Steps:**

1. Launch the application Visual Studio Code Service.
2. Observe that two scroll bar appearing for whole page at 400% zoom.

**Actual:**
Two scroll bar appearing for whole page at 400% zoom.
**NOTE:**Same issue is observed for all the pages.

**Expected Result:**
There should be only one scroll bar for whole page at 400% zoom.

**User Impact:**
Low vision users who zoom the page for better visibility will find it difficult to view the content if two dimensional scroll bars are provided.

**Attachment:**
![A11y_Visual Studio Code Services_Home_Reflow](https://user-images.githubusercontent.com/97668775/149905321-8de6bdda-d9ea-4edd-8969-67957bb49c65.png)

**Comment by gregvanl** [comment]
Closing as duplicate of https://github.com/microsoft/vscode-docs/issues/3432. Behavior is based on browser not code.visualstudio.com

**Comment by poojapatil36** [comment]
As per discussion with SME this bug need to be fixed as the dropdown icon is not visible at 400% zoom as shown in below image.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/97668775/151743455-6e90351e-7374-43b1-a6c9-05e7a062d717.png)

Opening a new bug as this bug is not fixed.

**Comment by gregvanl** [comment]
@poojapatil36 The browser is displaying a scroll bar so that content outside the view can be brought in to the view.
I still do not understand the issue.  
Please stop creating new issues, you can reopen the closed issues if you feel they need to be fixed.

**Comment by poojapatil36** [comment]
@gregvanl for web page there should not be two scroll bar present, only one scroll bar should be there when we zoom page to 400%.

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Please find below attachment for reference.
Github tag: #Closed;
![5102_Fixed](https://user-images.githubusercontent.com/97668775/218078871-41f3e658-3b74-41c4-af3b-ab7879012760.gif)
